scripts/test-proxy-startup-with-env-vars.sh

```diff
@@ -2,24 +2,38 @@
 set -e  # Exit on any error
 
 ##### This script is meant to be run in ci #####
-# It tests that the env vars defined in .env.example.holesky are correct.
+# It tests that the env vars defined in the specified environment file are correct.
 # It starts the eigenda-proxy with those env vars, waits 5 seconds, and then kills the proxy.
-# If any deprecated flags are still being used in .env.example.holesky, the script will fail.
+# If any deprecated flags are still being used in the specified environment file, the script will fail.
+
+ENV_FILE=$1
+
+# Check if the environment file exists
+if [ ! -f "$ENV_FILE" ]; then
+    echo "Error: Environment file $ENV_FILE does not exist"
+    echo "Current working directory: $(pwd)"
+    echo "Files in current directory:"
```

Comment:
prob need to add a check in case someone forgets to pass argument. Then exit with usage comment

Reply:
[3387be2](https://github.com/Layr-Labs/eigenda-proxy/pull/349/commits/3387be2c8f0f794223723cbead35a8a835d5f5f9)